### PR TITLE
Updating Websocket 2.0 release notes

### PR DESCRIPTION
**Pull request by gabrielbussolo**
contributes to #395 and complements #414

### CODE REVIEW

File: install/websocket/bin/build.xml [hunk 1]
```diff
@@ -1,22 +1,4 @@
-<!--
```
Comment (edbratt) [inline install/websocket/bin/build.xml:1]: Please keep the license. Update the copyright date to include 2018, 2020.

File: install/websocket/docs/WebSocketTCK2.0-ReleaseNotes.html [hunk 2]
```diff
@@ -1,23 +1,5 @@
 <!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
-<html><head><title>Jakarta WebSocket TCK, Version 2.0 Release Notes, June 2020</title>
-<!--
```
Comment (edbratt) [inline install/websocket/docs/WebSocketTCK2.0-ReleaseNotes.html:3]: Please keep the license header

File: install/websocket/docs/index.html [hunk 3]
```diff
@@ -1,25 +1,7 @@
 <!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
 <html>
 <head>
-<!--
```
Comment (edbratt) [inline install/websocket/docs/index.html:4]: Please keep the license header. Update the 2nd date in copyright from 2019 to 2020.

**Comment by gabrielbussolo** [comment]
@edbratt done, thanks for your review!

**Pull request merged**